assessments/projects/TiKV/self-assessment.md

```diff
@@ -0,0 +1,363 @@
+# TiKV Security Self-Assessment
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Background](#background)
+  * [Architecture](#architecture)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+* [STRIDE Threat Model](#stride-threat-model)
+* [Questions about potential vulnerabilities](#questions-about-potential-vulnerabilities)
+* [Suggested action items](#suggested-action-items)
+
+## Metadata
+|   |  |
+| -- | -- |
+| Software | [TiKV](https://github.com/tikv/tikv/tree/master)  |
+| Security Provider | No  |
+| Languages | Rust, Java, Go, and C |
+| SBOM |Link to the dependencies file: [Cargo.toml](https://github.com/tikv/tikv/blob/master/Cargo.toml) |
```

Comment:
```suggestion
| SBOM |Link to the dependencies file: TiKV does not currently generate an SBOM on release |
```

The cargo manifest and software bill of materials are different types of items with differing purposes.